config/provisioners/natss/README.md

```diff
@@ -0,0 +1,47 @@
+# NATS Streaming Channels
+
+
+### Deployment steps:
+
+1. Setup [Knative Eventing](../../../DEVELOPMENT.md).
+1. Apply the 'natss' ClusterChannelProvisioner, Controller, and Dispatcher.
+     ```shell
+     ko apply -f config/provisioners/natss/
+     ````
+1. Create Channels that reference the 'natss'.
+
+    ```yaml
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Channel
+    metadata:
+      name: foo
+    spec:
+      provisioner:
+        apiVersion: eventing.knative.dev/v1alpha1
+        kind: ClusterChannelProvisioner
+        name: natss
+    ```
+
+### Components
+
+The major components are:
+* NATS Streaming
+* ClusterChannelProvisioner Controller
+* Channel Controller
+* Channel Dispatcher
+
+NATS Streaming is deployed as a StatefulSet. 
+For tuning NATS Streaming, see:
+https://github.com/nats-io/nats-streaming-server#configuring
```

Comment:
Also state the name of the `ConfigMap` that contains the configuration.

Reply:
Added in README.
Done